components/script/dom/xrframe.rs

```diff
@@ -79,6 +80,13 @@ impl XRFrameMethods for XRFrame {
         DomRoot::from_ref(&self.session)
     }
 
+    /// <https://www.w3.org/TR/webxr/#dom-xrframe-predicteddisplaytime>
+    fn PredictedDisplayTime(&self) -> Finite<f64> {
+        // TODO: If inline, return rAF callback time instead
```

Comment:
I think this is clearer(I didn't know there is a webxr specific raf until reading the spec just now): "return the same value as the timestamp passed to the XRFrameRequestCallback"